package.json

```diff
@@ -1,133 +1,134 @@
-    "test:test-db": "NODE_ENV=test PGPASSFILE=/mnt/ramdisk_move_test/.pgpass TEST_DIR=\"db/**\" jest --runInBand",
-    "serve": "vue-cli-service serve",
-    "build": "vue-cli-service build"
-  },
-  "dependencies": {
-    "@hapi/boom": "^8.0.1",
-    "@hapi/cookie": "^10.1.2",
-    "@hapi/crumb": "^7.3.2",
-    "@hapi/good": "^8.2.4",
-    "@hapi/good-console": "^8.1.2",
-    "@hapi/good-squeeze": "^5.2.1",
-    "@hapi/hapi": "^18.4.0",
-    "@hapi/joi": "^15.1.1",
-    "@hapi/scooter": "^5.1.1",
-    "@sendgrid/mail": "^6.4.0",
-    "aws-sdk": "^2.566.0",
-    "blankie": "^4.1.1",
-    "csv-generate": "^3.2.3",
-    "csv-stringify": "^5.3.3",
-    "d3-array": "^2.3.3",
-    "d3-axis": "^1.0.12",
-    "d3-dsv": "^1.2.0",
-    "d3-format": "^1.4.1",
-    "d3-scale": "^3.2.0",
-    "d3-selection": "^1.4.0",
-    "d3-selection-multi": "^1.0.1",
-    "esm": "^3.2.25",
-    "file-saver": "^2.0.2",
-    "jsdom": "^15.2.1",
-    "luxon": "^1.21.1",
-    "mapbox-gl": "^1.5.0",
-    "module-alias": "^2.2.2",
-    "mustache": "^3.1.0",
-    "nodemailer": "^6.3.1",
-    "openid-client": "^2.4.5",
-    "pdfmake": "^0.1.62",
-    "pg-promise": "^9.3.5",
-    "request": "^2.88.0",
-    "request-promise-native": "^1.0.8",
-    "uuid": "^3.3.3",
-    "v-calendar": "^0.9.7",
-    "vue": "^2.6.3",
-    "vue-router": "^3.1.3",
-    "vuelidate": "^0.7.4",
-    "vuex": "^3.1.0",
-    "whatwg-fetch": "^3.0.0"
-  },
-  "devDependencies": {
-    "@babel/core": "^7.7.2",
-    "@vue/cli-plugin-babel": "^3.12.1",
-    "@vue/cli-plugin-eslint": "^3.12.1",
-    "@vue/cli-service": "^3.12.1",
-    "@vue/eslint-config-airbnb": "^4.0.0",
-    "@vue/test-utils": "^1.0.0-beta.29",
-    "babel-core": "^7.0.0-bridge.0",
-    "babel-eslint": "^10.0.3",
-    "babel-jest": "^24.9.0",
-    "documentation": "^12.1.3",
-    "eslint": "^5.16.0",
-    "eslint-import-resolver-alias": "^1.1.2",
-    "eslint-plugin-import": "^2.18.2",
-    "eslint-plugin-vue": "^5.1.0",
-    "jest": "^24.9.0",
-    "jest-serializer-vue": "^2.0.2",
-    "jest-transform-stub": "^2.0.0",
-    "lint-staged": "^9.4.2",
-    "npm-run-all": "^4.1.5",
-    "postcss-nested": "^4.2.1",
-    "stylelint": "^11.1.1",
-    "stylelint-config-standard": "^19.0.0",
-    "vue-jest": "^3.0.5",
-    "vue-template-compiler": "^2.6.3",
-    "webpack-bundle-analyzer": "^3.6.0"
-  },
-  "gitHooks": {
-    "pre-commit": "npm-run-all pre-commit:*"
-  },
-  "lint-staged": {
-    "lib/**/*.js": [
-      "documentation lint"
-    ],
-    "*.js": [
-      "vue-cli-service lint"
-    ],
-    "*.postcss": [
-      "stylelint"
-    ],
-    "*.py": [
-      "pylint"
-    ],
-    "*.sh": [
-      "shellcheck --shell=bash"
-    ],
-    "*.vue": [
-      "vue-cli-service lint",
-      "stylelint"
-    ]
-  },
-  "_moduleAliases": {
-    "@": "."
-  },
-  "browserslist": [
-    "last 1 Firefox version",
-    "last 1 Chrome version",
-    "last 1 Edge version"
-  ]
-}
+{
```

Comment:
large diff is probably due to `npm version patch` fixing newlines in the process